mesonbuild/compilers/asm.py

```diff
@@ -27,6 +27,7 @@
 class NasmCompiler(Compiler):
     language = 'nasm'
     id = 'nasm'
+    links_with_msvc = False
```

Comment:
Class "variables" like these should really be constants. Since the value can change it should be an instance variable instead and only set in `__init__`.